src/index.ts

```diff
@@ -28,30 +28,44 @@
 //     unmarshalled from an incoming request.
 
 import * as minimist from 'minimist';
+
 import {
   ErrorHandler,
   FunctionSignatureType,
   getServer,
   getUserFunction,
 } from './invoker';
 
+// Supported command-line flags
+const FLAG = {
+  PORT: 'port',
+  TARGET: 'target',
+  SIGNATURE_TYPE: 'signature-type', // dash
+};
+
+// Supported environment variables
+const ENV_VARIABLE = {
+  PORT: 'FUNCTION_PORT',
+  TARGET: 'FUNCTION_TARGET',
+  SIGNATURE_TYPE: 'FUNCTION_SIGNATURE_TYPE', // underscore
+};
+
 enum NodeEnv {
   PRODUCTION = 'production',
 }
 
 const argv = minimist(process.argv, {
-  string: ['port', 'function-target', 'function-signature-type'],
+  string: [ENV_VARIABLE.PORT, FLAG.TARGET, FLAG.SIGNATURE_TYPE],
 });
 
 const CODE_LOCATION = process.cwd();
-const PORT = argv['port'] || process.env.PORT || '8080';
+const PORT =
+  argv[ENV_VARIABLE.PORT] || process.env[ENV_VARIABLE.PORT] || '8080';
 const FUNCTION_TARGET =
```

Comment:
While I can see how the refactoring you are proposing makes sense, it also makes the code a bit less readable, in my opinion, because of the extra mappings. For example, we are mapping `FUNCTION_TARGET` string to `ENV_VARIABLE.TARGET` and then resolving it to `FUNCTION_TARGET` const.

If the names without 'function' prefix are to be the canonical ones, then let's drop `FUNCTION_` prefix from `FUNCTION_TARGET`, `FUNCTION_SIGNATURE_TYPE_STRING`, `FUNCTION_SIGNATURE_TYPE` consts.

Reply:
Dropped the `FUNCTION_` prefix.